app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -427,7 +427,9 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
 
         override fun onDrawerClosed(drawerView: View) {
           super.onDrawerClosed(drawerView)
-          fragment.activity!!.invalidateOptionsMenu()
+          // It's possible in some rare cases for the activity to be gone while the drawer is
```

Comment:
Ah, if you have the context for it I think it's worth filing an issue explaining how to repro it. I found it completely accidentally and couldn't repro it again (I fixed it entirely based on the stack trace & reasoning). A consistent repro may allow us to possibly write a test for it (though Robolectric differs from Android most noticeably with activity lifecycle behaviors, so it may not be possible to test in a non-convoluted way).

Reply:
Okay, I will file the issue.